SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -245,16 +246,68 @@ def setBC(self):
                     alpha_ym, alpha_yp = temp_ym, temp_yp
                     alpha_zm, alpha_zp = temp_zm, temp_zp
 
-                    beta_xm, beta_xp = temp_xm*0, temp_xp*0
-                    beta_ym, beta_yp = temp_ym*0, temp_yp*0
-                    beta_zm, beta_zp = temp_zm*0, temp_zp*0
+                    beta_xm, beta_xp = temp_xm*1, temp_xp*1
```

Comment:
looks like you are changing defaults here??

Reply:
@sgkang: this is intentional?? Just checking

Reply:
no it seems my mistake. thanks @lheagy !